pkg/store/cache/caching_bucket_factory.go

```diff
@@ -87,6 +87,16 @@ func NewCachingBucketFromYaml(yamlContent []byte, bucket objstore.Bucket, logger
 	}
 
 	var c cache.Cache
+	cfg := cache.NewCachingBucketConfig()
+
+	// Configure cache.
+	cfg.CacheAttributes("chunks", c, isTSDBChunkFile, config.ChunkObjectAttrsTTL)
```

Comment:
Let's pass `nil` directly to clearly indicate to the reader that we are only configuring paths here:
```suggestion
	cfg.CacheAttributes("chunks", nil, isTSDBChunkFile, config.ChunkObjectAttrsTTL)
```
```suggestion
	cfg.CacheAttributes("chunks", c, isTSDBChunkFile, config.ChunkObjectAttrsTTL)
```